pennylane_lightning/core/src/simulators/lightning_kokkos/measurements/MeasurementsKokkos.hpp

```diff
@@ -506,11 +507,18 @@ class Measurements final
      * @return Floating point std::vector with probabilities.
      * The basis columns are rearranged according to wires.
      */
-    std::vector<PrecisionT> probs(const std::vector<std::size_t> &wires) {
+    auto probs(const std::vector<std::size_t> &wires)
+        -> std::vector<PrecisionT> {
         PL_ABORT_IF_NOT(
             std::is_sorted(wires.cbegin(), wires.cend()),
             "LightningKokkos does not currently support out-of-order wire "
             "indices with probability calculations");
+
+        // If all wires are requested, dispatch to `this->probs()`
+        if (wires.size() == this->_statevector.getNumQubits()) {
```

Comment:
Do we want to also defer in this case?
```suggestion
        if (wires.empty() || wires.size() == this->_statevector.getNumQubits()) {
```